nexus/reconfigurator/planning/src/blueprint_editor/sled_editor/underlay_ip_allocator.rs

```diff
@@ -0,0 +1,220 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file was moved from `blueprint_builder/ip_allocator.rs`, but changed enough (and is small enough) that it's probably worth a review as though it were new.